packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceProjectCreate.test.ts

```diff
@@ -93,6 +94,7 @@ describe('Force Project Create', () => {
       inputBoxSpy.onCall(0).returns(undefined);
       inputBoxSpy.onCall(1).returns('');
       inputBoxSpy.onCall(2).returns(PROJECT_NAME);
+      inputBoxSpy.onCall(3).returns(PROJECT_NAME_WITH_LEADING_TRAILING_SPACES);
```

Comment:
my apologies as I know you're following the pattern..but having a single before/after here and depending on state in each of the tests leads towards us having a bad time.  Would you mind changing this to a beforeEach/after Each and just assign the returns for each test in each it?   So it'd be something like 
```
beforeEach(() => { 
  inputBoxSpy = sinon.stub(vscode.window, 'showInputBox');
});

afterEach(() => {
  inputBoxSpy.restore();
});

....

    it('Should return Continue with trimmed project name if project name input has leading and or trailing spaces', async () => {
      inputBoxSpy.returns(PROJECT_NAME_WITH_LEADING_TRAILING_SPACES);
      const gatherer = new SelectProjectName();
...
```

Reply:
Just took a crack at this.  Let me know how things look!